apps/web/src/pages/integrations/components/UpdateIntegrationCommonFields.tsx

```diff
@@ -24,19 +31,21 @@ export const UpdateIntegrationCommonFields = ({ provider }: { provider: IIntegra
 
   return (
     <>
-      <Controller
-        control={control}
-        name="active"
-        defaultValue={false}
-        render={({ field }) => (
-          <Switch
-            checked={field.value}
-            label={field.value ? 'Active' : 'Disabled'}
-            data-test-id="is_active_id"
-            {...field}
-          />
-        )}
-      />
+      <When truthy={showActive}>
```

Comment:
Add this because we should not allow user to activate and deactivate a novu provider

Reply:
I guess it is by design, but i am not sure it is the best way going forward, for a couple of reasons:
* there could be a case where the user won't want to use `Novu provider` due user's internal business interests (the easy way to handle it is to deactivate it).
* especially in the integration conditions that we will add there could be a case where the user won't want to have Novu as an optional provider as a backoff opportunity (the easy way to handle it is to deactivate it).
* use a specific provider (and not Novu's) even if it is failing.
* still having if statements in the code related to the Novu provider. 😅

Reply:
We only allow using the Novu provider if they do not have any other provider for the same channel so this should not be a case.

Reply:
Actually, this is the problem i had in my mind but it is the current behavior and not part of this scope, them never mind for not :)